micro-benchmarks/DRB171-default-orig-no.c

```diff
@@ -0,0 +1,43 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+/**
+ * MG.C: This file is part of kernel of the NAS Parallel Benchmarks 3.0 MG suit.
+ * Tsan and Intel Inspector can not correctly analysis the default(share), and report a false postive.
```

Comment:
analysis is a noun. analyze is the verb.   How do you know default(shared) cause throuble to the tools? Would removing default(shared) make a difference?